package.json

```diff
@@ -68,7 +68,10 @@
   },
   "dependencies": {
     "@emotion/core": "10.0.22",
+    "@emotion/css": "10.0.22",
```

Comment:
Isn't this part of `@emotion/core`? 🤔

Reply:
otherwise we will get unresolved dependency errors as rollup compiles the code to this

```
import _styled from '@emotion/styled-base';
...
import _css from '@emotion/css';
```

otherwise we can add them to the "external" array